clipmenud

```diff
@@ -85,12 +85,15 @@ kill_background_jobs() {
     # While we usually _are_, there are no guarantees that we're the process
     # group leader. As such, all we can do is look at the pending jobs. Bash
     # avoids a subshell here, so the job list is in the right shell.
-    local bg
-    bg=$(jobs -p)
+    local -a bg
+
+    readarray -t bg < <(jobs -p)
 
     # Don't log `kill' failures, since with KillMode=control-group, we're
     # racing with init.
-    [[ $bg ]] && kill -- "$bg" 2>/dev/null
+    for i in ${!bg[@]}; do
+        kill -- ${bg[i]} 2>/dev/null
+    done
```

Comment:
Why do all this instead of just

    kill -- "${bg[@]}" 2>/dev/null

?

Reply:
No particular reason... looking around in `clipmenud` I saw a `while` looping an array, so I kind of copycatted it...